### ISSUE TITLE
Faucheur d'âme Roi Liche

### ISSUE THREAD

**Issue by Wilpok** [way-of-elendil/3.3.5#1205]
**Description**
La résistance ombre est inversée sur le débuff du faucheur d'âme, officiellement, seulement le 1er tick (à la pose) est résisitible via la résistance ombre est le second non sur elendil c'est le contraire, le 1er n'est pas résistible et le second si.
**Étapes de reproduction**
Fournissez les informations nécessaires pour reproduire le problème. Par exemple
- Citadelle de la couronne de glace
- 10 nm, 10hm, 25nm, 25hm
- Le roi liche
- Faucheur d'âme à partir de la 2nde phase (< 70% jusqu'au down)
**Comportement attendu**
Le faucheur d'ame n'est pas censé se voir appliquée la resistance à l'ombre sur le 2eme tick mais sur le 1er. ici c'est inversé.
**Captures d’écran / Vidéos**
![image](https://user-images.githubusercontent.com/65713751/111632503-d4639c80-87f4-11eb-96d5-8ad42a56a718.png)
(Sur elendil)
![image](https://user-images.githubusercontent.com/65713751/111632552-e1808b80-87f4-11eb-9434-e60c9bb78b71.png)
(Sur officiel)
Lien des logs en question : http://www.worldoflogs.com/reports/li4fd7jabtswxtk8/log/?s=9757&e=10675 (elendil)
http://www.worldoflogs.com/reports/o6ur0qdk3r7nx8jn/log/?s=1711&e=2239 (Officiel)
Peut être certains penseront que c'est mineur mais non car le 2eme tick est bien plus important que le 1er (du 90k en 25hm) tandis que le 1er est vraiment négligeable.
**Ressources additionnelles**
https://fr.wowhead.com/spell=69409/faucheur-d%C3%A2me#english-comments
Cf encore d'autres logs dans les commentaires

**Comment by Ischjgt** [comment]
https://github.com/way-of-elendil/3.3.5/issues/847

**Comment by Wilpok** [comment]
ça ne change pas le fait que le 2ème tick est résisté alors qu'il n'est pas censé l'être, mais j'peux le rajouter sur l'autre ticket si tu préfères.

**Comment by joshwhedon** [comment]
Non, le coup de depart est une attaque de melee, pas une attaque d'ombre, donc la resistance ne s'applique pas (l'armure oui).
Le dot inflige des degats d'ombre donc ca s'applique sur le dot.
J'ai aussi du mal a comprendre en quoi le log appuie ton argument

**Comment by Wilpok** [comment]
Non le 2eme tick ne prends pas en compte la resistance justement uniquement la réduction flat de degats, (comme ce qui est montré sur les logs d'offi pas de "(R:...)" sur ce tick tandis que sur le 1er oui

**Comment by joshwhedon** [comment]
![image](https://user-images.githubusercontent.com/543699/111701995-bdbe5380-888f-11eb-8564-94cd8771d96e.png)
Dans ton image, le premier "tick" c'est avant le afflicted by ... C'est a dire avant que le dot soit applique. Et effectivement pas de R: ...

**Comment by Wilpok** [comment]
Mais le tick des 50k n'est pas censé être resistable c'est ce que je te dis ^^ comme sur offi regarde les degats de reaper que prend le tank sur offi 5secondes après la pose, aucun n'est resist alors que sur elendil si, sur elendil y'a une resist meme sur l'auto attaque

**Comment by joshwhedon** [comment]
Pardon j'ai cru que les logs etaient tous de l'offi.

**Comment by joshwhedon** [comment]
Possible que le R soit en fait la reduction par l'armure plutot que la reduction par la resistance dans les logs ?

**Comment by Wilpok** [comment]
Non non R c'est resistance A c'est absorb, y'a pas de chose liée a l'armure ^^ juste les dégats sont plus bas quoi sur les auto attaques

**Comment by Wilpok** [comment]
D'ailleurs Josh : Pourquoi le 1er tick est résistable : https://www.mmo-champion.com/threads/695843-Lich-King-Encounter-Soul-Reaper

"The application of Soul Reaper in 25 man (read: the second you get the debuff), he also deals 50% weapon damage as Shadow (not reduced by armor) which can be up to 40k. This along with a melee hit can kill any tank if they are not aware and have cooldowns for it along with cooldowns of your priests and paladins. And this is basically the reply. Rotate cooldowns when the Soul Reaper timer is up AND taunt after the tank has the debuff. The 60k damage after 5 seconds is not what kills the tanks if they taunt properly but the initial shadow damage."

(renote bien que l'armure ne marche sur aucun des 2 ticks du coup, seulement la resistance est applicable au 1er tick)

**Comment by Wilpok** [comment]
(ce qui est confirmé par les logs d'ailleurs)

**Comment by Wilpok** [comment]
J'ose un up savoir si y'a un truc ou pas d'solution ou qu'il manque des preuves? ^^

**Comment by joshwhedon** [comment]
Surtout une question de temps, et d'avoir aucune idee de comment regler le probleme sans en causer d'autres ailleurs.

**Comment by Wilpok** [comment]
P'tit up savoir si y'a des pistes ou pas, car atm lk est toujours free entre ça et les degats réduits des autos (sur tous les raids et boss), et les pop esprits/faucheur qui disparaissent etc...

**Comment by joshwhedon** [comment]
Pour etre parfaitement honnete j'ai vraiment pas le temps de bosser sur woe ces temps ci.

**Comment by ltbastion** [comment]
Petit up pour savoir si une personne du staff est dessus ? :)

**Comment by Endalaust** [comment]
Passé avec https://discord.com/channels/280446236035776512/697715145325150268/903777508355342366

**Comment by Wilpok** [comment]
On réouvre dcp vu qu'il marche pas?

**Comment by Wilpok** [comment]
> Après pas mal de recherches, je suis du même avis que Josh. Je ne vois pas comment régler le problème de façon propre et en correspondant parfaitement à la demande. De ce fait, je vais plutôt proposer une désactivation totale des résistances sur ce sort. L'effet indésirable est que la résistance ne s'appliquera plus non-plus sur l'attaque, mais comme mentionné plus haut, l'effet est mineur pour un coup qui fait à peu près 14k de dégâts sur un tank.



Salut @Kaasss-woe,

C’est tjrs d’actualité de passer une update qui enlève les resi au soul reaper pour que ca devienne plus correct? 🙂

**Comment by Wilpok** [comment]
Si jamais je remets des informations supplémentaires : 

![image](https://github.com/user-attachments/assets/62059c73-7fc4-44a1-9248-11026ac7038c)
Par rapport à ce que josh avait dit, en gros la mécanique du soulreaper sur le 1er tick prend la resistance en compte car c'est des dégats soit physique soit magiques (résistables) donc cela prend la stat qui diminue le - les degats et tape dessus donc en général (même toujours) comme c'est sur tank le 1er tick appliqué est en dégats magique résistables et diminué par la rési ombre (comme on peut le voir sur les toujours) alors que le 2nd tick est uniquement du flat 70k dégats uniquement mitigé par les réduction dégat, comme on le voit sur logs de l'époqué confirmé/coroboré par des logs de classic 
cf 
![image](https://github.com/user-attachments/assets/6cf0614c-f2a8-44c3-a0c9-923534132ca4)
 ou on voit bien que uniquement la mitigation est présente (M: ) selon les buff appliqués au tank( que l'on voit au tank) 

et également je pense (rien de confirmé encore je dois faire de nouveaux test) que les weapon damage sont bas par rapport à ce qu'ils devraient être (ou prennent l'armure en compte ou armure + rési aucune idée je dois recheck ça )

**Comment by MissZay** [comment]
le tick de dégât initial inflige 50% WDM donc si les dégâts de base du boss sont plus bas que la normale le premier tick du soul reaper sera réduit d'autant

**Comment by Wilpok** [comment]
Oui je sais c’est pour ça que j’ai dit que j’allais le tester, mais j’ai pas encore eu le temps
